modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -452,8 +473,10 @@ public void processOpts() {
         }
 
 
-        if (!this.apiFirst && !this.reactive) {
-            additionalProperties.put("useSpringfox", true);
+        if (this.useSpringfox) {
+            if (!this.apiFirst && !this.reactive) {
```

Comment:
minor, can simplify to: this.useSpringFox && !this.apiFirst && !this.reactive

Reply:
I'll add comments - I assume that springFox cannot be used when apiFirst and reactive is enabled but we don't use those options so I wanted it to be clear that this check is separate from the `useSpringFox` option check